addons/hide-backpack/addon.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "Hide backpack",
```

```suggestion
  "name": "Replace backpack bar with button",
```
Can someone come up with something better?